test/src/test/java/hudson/model/RunParameterDefinitionTest.java

```diff
@@ -49,6 +52,14 @@
     @Rule
     public JenkinsRule j = new JenkinsRule();
 
+    @Issue("JENKINS-47530")
+    @Test public void returnNullOnNullProject() throws Exception {
```

Comment:
It's also not required to use Mockito AFAICT. You can just pass a non-existent `projectName` in the constructor.

Reply:
Okay, not using Mockito to trigger the issue required me to add an additional null check to `jenkins/model/Jenkins.java`, because StringTokenizer will fail to initialize if passed `null` as the first constructor argument. see updated PR